store/copr/coprocessor.go

```diff
@@ -303,6 +307,7 @@ type buildCopTaskOpt struct {
 	eventCb  trxevents.EventCallback
 	respChan bool
 	rowHints []int
+	elapsed  *time.Duration
```

Comment:
Why is it optional?

Reply:
The duration of tasks building by region error is ignored here, because I only want to calculate how long it takes before sending requests.